bionic/flow.py

```diff
@@ -1900,6 +1900,7 @@ def core__process_executor(
 
     builder.assign("core__aip_execution__enabled", False, persist=False)
     builder.assign("core__aip_execution__gcp_project_name", None, persist=False)
+    builder.assign("core__aip_execution__docker_build_enabled", False, persist=False)
```

Comment:
WDYT about removing this entity and making the build process happen automatically if no image name is provided?

Reply:
Good question. I was initially thinking that if `core__aip_execution__docker_image_uri` is specified and `core__aip_execution__docker_build_enabled` is true, then maybe we can build a docker image with the tag specified in `core__aip_execution__docker_image_uri`.